addons/web/static/tests/chrome/menu_tests.js

```diff
@@ -0,0 +1,55 @@
+odoo.define('web.menu_tests', function (require) {
+    "use strict";
+
+    const testUtils = require('web.test_utils');
+    const Menu = require('web.Menu');
+    const SystrayMenu = require('web.SystrayMenu');
+    const Widget = require('web.Widget');
+
+
+    QUnit.module('chrome', {}, function () {
+        QUnit.module('Menu');
+
+        QUnit.test('Systray on_attach_callback is called', async function (assert) {
+            assert.expect(4);
+
+            const parent = testUtils.createParent({
+                debug: true,
+                data: {
+                    'ir.ui.view': {
+                        fields: [],
+                        check_access_rights: () => true,
```

Comment:
i think you only need this for the debug manager, so as you override SystrayMenu.Items, it should not be necessary (maybe it was at some point of your development?)